test/integration/targets/vmware_folder_info/tasks/main.yml

```diff
@@ -0,0 +1,78 @@
+# Test code for the vmware_folder_info module.
+# Copyright: (c) 2019, David Hewitt (@davidmhewitt) <davidmhewitt@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Wait for Flask controller to come up online
```

Comment:
Could you please reformat the testcase since there has been a change in vcsim testcases ? You can take a look at [vmware_about_facts](https://github.com/ansible/ansible/tree/devel/test/integration/targets/vmware_about_facts) as a reference.